src/lifetimes.md

```diff
@@ -1,9 +1,14 @@
 # Lifetimes
 
 Rust enforces these rules through *lifetimes*. Lifetimes are effectively
-just names for scopes somewhere in the program. Each reference,
-and anything that contains a reference, is tagged with a lifetime specifying
-the scope it's valid for.
+just names for areas of code somewhere in the program (you could also look at
+them as time intervals, except that the beginning and end of the time is denoted
+by when a line of code executes). Each reference, and anything that contains a
+reference, is tagged with a lifetime specifying the time it's valid for.
```

Comment:
I'm not sure I like this time metaphor. A more apt one might be "paths of execution"?